release/build_release.sh

```diff
@@ -199,4 +200,12 @@ function print_variables {
 	echo ""
 }
 
+function create_release_properties_file {
+	(
+		echo "build.timestamp=${_BUILD_TIMESTAMP}"
```

Comment:
```
function update_release_info_date {
	lc_cd "${_PROJECTS_DIR}"/liferay-portal-ee

	sed -i -e "s/release.info.date=.*/release.info.date=$(date +"%B %d, %Y")/" release.properties
}```

Later there is running the above function. Does it not conflict with this one? Or is a different one `release.properties` file?

Reply:
This isn't the same release.properties file. This is the "liferay-portal-ee" repo's file, but ours is in the "liferay-docker" repo's file.